src/main/resources/db/migration/schema/V20210203175529__create_table_user.sql

```diff
@@ -0,0 +1,14 @@
+CREATE SEQUENCE user_sequence INCREMENT BY 1;
+
+Create table tb_user(
+	id bigint DEFAULT nextval('user_sequence'),
```

Comment:
usar int8 para Longs e int4 para inteiros

Reply:
usar PRIMARY KEY para o id: https://github.com/EmersonDantas/S-W-R-Social-Network-API/blob/master/src/main/resources/db/migration/schema/V20200627152100__create_rebel.sql

Reply:
definir o login como UNIQUE, e o email tbm (se for o caso, oq deve ser)